assets/messages.yml

```diff
@@ -1054,6 +1054,27 @@ messages:
         This is a technical support issue; this site is for bug reports only. We do not have the resources to provide you with technical support.
         Please contact the *[Community Support|https://discord.gg/58Sxm23]*.
 
+        %quick_links%
+      fillname: []
+  temporary-outage:
+    - project:
+        - mc
+        - mcd
+        - mcl
+        - mcpe
+        - realms
+      name: Temporary Service Outage
+      shortcut: outage
+      message: |-
+        *Thank you for your report!*
+       
+        Issues such as these are often due to a temporary service outage, and are handled by the team outside the bug tracker.
+        
+        Watch [Mojang Status|https://twitter.com/MojangStatus] for official announcements, or [Downdetector|https://downdetector.com/status/minecraft] for community reported outages.
+        
+        If there is no general outage, you are free to create a new report, after searching for duplicate reports. However, please first try to verify that the problem is not with your local connection or network settings. Visit *[Minecraft Community Support|https://discord.gg/58Sxm23]* for assistance.
```

Comment:
Do we really want to make them re-report in that case?
Imo a comment would be sufficient, so we can reopen the report.

Reply:
Basically this would be the same process as Incomplete resolution, though in this case it would be "invalid". We could reopen if someone actually notices a comment on a resolved issue, but usually that does not happen. We also do the same for "crash logged automatically" where if the user can provide enough details (like the one with dispensing bonemeal on MCPE) then it's kept as a valid report.

Reply:
I'd also want a way to request bulk resolve these, and hopefully Arisa would put the message in. There's probably 50 open tickets on REALMS (and some MCPE) about not being able to connect due to various outages. I've just been ignoring those pending having this message.

Reply:
I think the process should be like for any other incorrectly closed ticket.
Users can also request on reddit or Discord.

Reply:
@MikeUnverferth Once the message is decided on we can do the bulk change like any other, just compile a comma separated list and I can do that one if you want to send it through to me. The comment can be done at the same time as part of the closure.

Reply:
Regarding the message, I'd almost suggest we recommend they contact Mojang support for all connectivity issues preventing access to a paid realm rather than re-opening a new bug. The only time I think it qualifies as a bug would be if it was an obscure reliable repro of how to bug it out, not "my realm can't be accessed at all".

My opinion is those sorts of things don't belong on the tracker, anything other than a mass outage we have no way of helping with and because its a paid service they really should be contacting Mojang given all the laws around providing services that are paid for etc.

Reply:
Seems reasonable. I adjusted the message accordingly.

Reply:
Sounds reasonable to me too. But we might want to ask Jay first, what do you think?

Reply:
Sure. I can copy the latest proposed message to slack for comment.

Reply:
Jay said it "looks good".